content/agents/installation.md

```diff
@@ -67,7 +67,9 @@ the download script is executed. It downloads the installation script
 from the temporary link and executes it.
 To use the `init_script` method, the IaaS provider and Cloudify plugin
 need to support userdata. Currently, the Openstack
-and AWS plugins support this installation method.
+and AWS plugins support this installation method. In addition, the image in
+use must already have Python 2.7.x installed and available through the
+system's `PATH`.
```

Comment:
What about GCP?

Reply:
@EarthmanT I just checked and it seems that GCP also supports it. Can you confirm?

Reply:
@EarthmanT What do you say, Trammell?

Reply:
GCP and Azure support it.

Reply:
@isaac-s Please update accordingly.